packages/shared/src/devBrowser.ts

```diff
@@ -19,20 +19,48 @@ export function setDevBrowserJWTInURL(url: URL, jwt: string): URL {
   return resultURL;
 }
 
-// Gets the dev_browser JWT from either the hash or the search
-// Side effect:
-// Removes dev_browser JWT from the URL as a side effect and updates the browser history
-export function getDevBrowserJWTFromURL(url: URL): string {
```

Comment:
🔧 This is a `@clerk/shared` public API method. Renaming it is a breaking change for the shared package. I would expect a major bump or at least a minor.

Reply:
1. We're treating the `shared` package as an internal, low-level package
2. This change goes into `main` so this is the v5 branch anyway
3. The packages use fixed versions, meaning that you cannot have a Clerk package that depends on `@clerk/shared@^5` (notice the caret)